content/docs/1.6.0/references/helm-values.md

```diff
@@ -182,8 +182,8 @@ Please also refer to this document [ocp-readme](https://github.com/longhorn/long
 | annotations | `{}` | Annotations to add to the Longhorn Manager DaemonSet Pods. Optional. |
 | enableGoCoverDir | `false` | Enable this to allow Longhorn to generate code coverage profiles |
 | enablePSP | `false` | For Kubernetes < v1.25, if your cluster enables Pod Security Policy admission controller, set this to `true` to ship longhorn-psp which allow privileged Longhorn pods to start |
-| helmPreUpgradeCheckerJob.enabled | `true` |  |
-| metrics.serviceMonitor.enabled | `false` |  |
+| helmPreUpgradeCheckerJob.enabled | `true` | Enable this to allow Longhorn to do pre-upgrade checks. Disable this to support ArgoCD. |
+| metrics.serviceMonitor.enabled | `false` | Enable this to create the ServiceMonitor |
```

Comment:
```suggestion
| metrics.serviceMonitor.enabled | `false` | Setting that allows the creation of ServiceMonitor. |
```

Reply:
Please also update https://longhorn.io/docs/1.6.0/monitoring/prometheus-and-grafana-setup/#install-longhorn-servicemonitor to make it optional if users deploy Longhorn with this setting enabled.

Reply:
> Setting that allows the creation of a Prometheus ServiceMonitor resource for longhorn-manager components. 

@jillian-maroket @mantissahz